pkg/proxy/util/nfacct/nfacct_linux.go

```diff
@@ -0,0 +1,307 @@
+//go:build linux
+// +build linux
+
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nfacct
+
+import (
+	"bytes"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"io"
+	"syscall"
+
+	"github.com/vishvananda/netlink/nl"
+	"golang.org/x/sys/unix"
+)
+
+// MaxLength represents the maximum length allowed for the name in a nfacct counter.
+const MaxLength = 31
+
+// nf netlink nfacct commands, these should strictly match with the ones defined in kernel headers.
+// (definition: https://github.com/torvalds/linux/blob/v6.7/include/uapi/linux/netfilter/nfnetlink_acct.h#L9-L16)
+const (
+	// NFNL_MSG_ACCT_NEW
+	cmdNew = 0
+	// NFNL_MSG_ACCT_GET
+	cmdGet = 1
```

Comment:
I can see 3 commands here https://nftables.org/projects/libnetfilter_acct/doxygen/html/group__nlmsg.html , are this related?

Reply:
We don't need the third one `NFNL_MSG_ACCT_GET_CTRZERO`, it does get + atomic reset to zero.